src/Lean/Environment.lean

```diff
@@ -423,22 +423,51 @@ structure ImportM.Context where
 
 abbrev ImportM := ReaderT Lean.ImportM.Context IO
 
-/-- An environment extension with support for storing/retrieving entries from a .olean file.
-   - α is the type of the entries that are stored in .olean files.
-   - β is the type of values used to update the state.
-   - σ is the actual state.
-
-   Remark: for most extensions α and β coincide.
-
-   Note that `addEntryFn` is not in `IO`. This is intentional, and allows us to write simple functions such as
-   ```
-   def addAlias (env : Environment) (a : Name) (e : Name) : Environment :=
-   aliasExtension.addEntry env (a, e)
-   ```
-   without using `IO`. We have many functions like `addAlias`.
-
-   `α` and ‵β` do not coincide for extensions where the data used to update the state contains, for example,
-   closures which we currently cannot store in files. -/
+/--
+An environment extension with support for storing/retrieving entries from a .olean file.
+ - α is the type of the entries that are stored in .olean files.
+ - β is the type of values used to update the state.
+ - σ is the actual state.
+
+For most extensions, α and β coincide. `α` and ‵β` do not coincide for extensions where the data
+used to update the state contains, for example, closures which we currently cannot store in files.
```

Comment:
```suggestion
used to update the state contains elements which cannot be stored in files (for example, closures) .
```

The "for example" here feels like it breaks up the flow, and this seems like a good time to suggest a change.

Reply:
Thanks!